README.md

```diff
@@ -15,15 +15,15 @@ XenCrypt is a module to help you set up encryption between XenIssuing and your a
 To be able to use XenCrypt, you will need to use a private key provided by Xendit.
 
 It includes several methods:
-- `generateSessionId` will encrypt a session key randomly generated used for symmetric encryption with Xenissuing.
+- `generateSessionId` will encrypt a session key randomly generated used for asymmetric encryption with Xenissuing.
 - `encrypt` would be used when setting sensitive data.
 - `decrypt` would be used whenever receiving sensitive data from Xenissuing.
 
 ### Usage
 ```android
 import XenCrypt
 try {
-    val xenKey = Base64.encode("BASE64_ENCODED_KEY_PROVIDED_BY_XENDIT")
+    val xenKey = Base64.encode("BASE64_PUBLIC_KEY")
```

Comment:
we should probably specify that it's only the base64 encoded portion without headers and footers (maybe give an example of valid key)